provider/datagen/tests/verify-zero-copy.rs

```diff
@@ -42,26 +42,23 @@ pub mod test {
         // don't drop to avoid dhat from printing stats at the end
         core::mem::forget(dhat::Profiler::new_heap());
 
+        let data_root = std::path::Path::new(std::env!("CARGO_MANIFEST_DIR")).join("tests/data");
+
         // Actual data is only needed to determine included locales.
         let locale_provider = DatagenProvider::try_new(
-            {
-                use icu_datagen::options::*;
-                let mut options = Options::default();
-                options.locales =
-                    LocaleInclude::Explicit(icu_testdata::locales().into_iter().collect());
-                options
-            },
+            Default::default(),
```

Comment:
Question: you can do this because the default behavior is to enumerate the directories to find the locales?

Reply:
Yes, default behaviour is all locales.